### DISCUSSION TITLE
`@mantine/carousel` and probably others are not recognized as part of `group:mantineMonorepo`

### DISCUSSION THREAD

**Discussion by mcmxcdev** [renovatebot/renovate#20712] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

"Pending Checks" section lists `@mantine/carousel`  separately from the grouped monorepo v6 upgrade for some reason:

![image](https://user-images.githubusercontent.com/16797721/222430891-2ea18f1d-54c6-4b32-9310-1320d25d2c7d.png)

Dependencies used in `package.json`:
```
"@mantine/carousel": "^5.10.5",
"@mantine/core": "^5.10.5",
"@mantine/dates": "^5.10.5",
"@mantine/dropzone": "^5.10.5",
"@mantine/hooks": "^5.10.5",
"@mantine/next": "^5.10.5",
"@mantine/notifications": "^5.10.5",
```

Entry in `monorepo.ts` looks correct to me: https://github.com/renovatebot/renovate/blob/c5edc5d54edaed93084b27026a7adf9c52e79296/lib/config/presets/internal/monorepo.ts#L132

### Relevant debug logs

`renovate.json`:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    "schedule:monthly",
    "helpers:disableTypesNodeMajor"
  ],
  "labels": ["dependencies"],
  "enabledManagers": ["npm", "github-actions", "dockerfile", "docker-compose"],
  "ignoreDeps": ["googleapis", "node", "pnpm", "postgres"],
  "rangeStrategy": "bump",
  "packageRules": [
    {
      "matchUpdateTypes": ["major"],
      "stabilityDays": 30
    },
    {
      "groupName": "all non-major dependencies",
      "groupSlug": "all-minor-patch",
      "matchPackagePatterns": ["*"],
      "matchUpdateTypes": ["minor", "patch"],
      "stabilityDays": 3
    },
    {
      "matchPackagePatterns": ["^@nrwl/", "^nx$"],
      "enabled": false
    }
  ]
}

```

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
This package does not populate the `repository` field in its registry data.

Compare https://registry.npmjs.org/@mantine/carousel to https://registry.npmjs.org/@mantine/core

In future please pay closer attention to 
![image](https://user-images.githubusercontent.com/6311784/222441405-03c189f4-f7fe-459e-b560-92a4c14f8ede.png)

I will convert this to a Discussion, future invalid issues will be deleted

**Reply by mcmxcdev** [reply]
I had to create it quickly and didn't pay attention, my bad!

**Comment by rarkins** [comment, marked as answer]
- https://github.com/mantinedev/mantine/pull/3652
